iree/tools/CMakeLists.txt

```diff
@@ -417,6 +419,20 @@ if(${IREE_BUILD_COMPILER})
     HOSTONLY
   )
 
+  iree_cc_binary(
+    NAME
+      ireec
```

Comment:
They are unique target names in unique directories, so no conflict. Yes, me need both (at least for the near future): the one in compiler-api links against the overall CAPI for the compiler (shared library) and is just a stub that invokes it. That layering is important for the deployment cases. Maybe we bring that approach back to iree/tools someday, but it would be a pretty big step.

Reply:
Okay, I hope we don't end up with two tools sharing a name and slightly different behavior that are both available on $PATH.

Reply:
Shouldn't happen. The one on the Python side is specifically for deployment in wheels. It *cannot* be built the same way on the iree/tools side -- it is built how it is on the Python side because of the specific way things are packaged there.

But they are identical (or will be once I do the swap out): They delegate to the same Main. It is just that the Python one gets that Main from a shared library that is also used for the in-process API.

Reply:
This does indeed seem to cause build system misbehavior: https://github.com/iree-org/iree/issues/9329